java-client/build.gradle.kts

```diff
@@ -205,6 +205,7 @@ dependencies {
     // For AwsSdk2Transport
     "awsSdk2SupportCompileOnly"("software.amazon.awssdk","sdk-core","[2.15,3.0)")
     "awsSdk2SupportCompileOnly"("software.amazon.awssdk","auth","[2.15,3.0)")
+    "awsSdk2SupportCompileOnly"("io.netty","netty-handler","4.1.108.Final")
```

Comment:
This dependency should not be needed, could you clarify why it is there?

Reply:
We see in our service that the AWS transport directly rethrows Netty exceptions, so in order to follow the catch-create-throw pattern from your implementation of the HC5 handling, we need access to the Netty dependency.